legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html

```diff
@@ -0,0 +1,21 @@
+<div class="no-page-break">
```

Comment:
This is not the final amalgamating corporation information, just a place holder.